public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/CreateNew/CreateNew.js

```diff
@@ -0,0 +1,474 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import { 
+  EuiTitle, 
+  EuiSpacer, 
+  EuiIcon, 
+  EuiText, 
+  EuiSwitch, 
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiAccordion,
+  EuiFormRow,
+  EuiFieldText,
+  EuiCheckbox,
+  EuiSelect,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiFieldNumber,
+  EuiCallOut
+ } from '@elastic/eui';
+import { EmbeddablePanel } from '../../../../../../../src/plugins/embeddable/public';
+import './styles.scss';
+import EnhancedAccordion from '../../EnhancedAccordion';
+
+function CreateNew({ embeddable, closeFlyout, core, services, index }) {
+  const [isShowVis, setIsShowVis] = useState(false);
+  const title = embeddable.getTitle();
+  const history = {
+    location: { pathname: '/create-detector', search: '', hash: '', state: undefined },
+    push: (value) => console.log('pushed', value),
+    goBack: closeFlyout,
+  };
+  const createMonitorProps = {
+    ...history,
+    history,
+    httpClient: core.http,
+    // This is not expected to be used
+    setFlyout: () => null,
+    notifications: core.notifications,
+    isDarkMode: core.isDarkMode,
+    notificationService: services.notificationService,
+    edit: false,
+    updateMonitor: () => null,
+    staticContext: undefined,
+    isMinimal: true,
+    defaultName: `${title} anomaly detector 1`,
+    defaultIndex: index,
+    defaultTimeField: embeddable.vis.params.time_field,
+    isDefaultTriggerEnabled: true,
+  };
+
+  const intervalOptions = [
+    { value: 'option_one', text: '10 minutes' },
+    { value: 'option_two', text: '1 minutes' },
+    { value: 'option_three', text: '5 minutes' },
+  ];
+  const [intervalValue, setIntervalalue] = useState(intervalOptions[0].value);
+  const intervalOnChange = (e) => {
+    setIntervalalue(e.target.value);
+  };
+
+  const delayOptions = [
+    { value: 'option_one', text: '10 minutes' },
+    { value: 'option_two', text: '1 minutes' },
+    { value: 'option_three', text: '5 minutes' },
+  ];
+  const [delayValue, setDelayValue] = useState(delayOptions[0].value);
+  const delayOnChange = (e) => {
+    setDelayValue(e.target.value);
+  };
+
+  const detectorNameFromVis = embeddable.vis.title + ' anomaly detector 1';
+  const featureList = embeddable.vis.params.series;
+  console.log("feature list: " + featureList)
+  console.log("feature name: " + featureList[0].label)
+
+  const anomaliesOptions = [
+    { value: 'option_one', text: 'Field value' },
+    { value: 'option_two', text: 'Custom expression' },
+  ];
+  const [anomaliesValue, setAnomaliesValue] = useState(anomaliesOptions[0].value);
+  const anomaliesOnChange = (e) => {
+    setAnomaliesValue(e.target.value);
+  };
+
+  const aggMethodOptions = [
+    { value: 'avg', text: 'AVG' },
+    { value: 'sum', text: 'SUM' },
+  ];
```

Comment:
We support more than just avg/sum, you can see existing create flow for what aggregations we support out of the box.